iphone/Maps/OMaps.plist

```diff
@@ -92,6 +92,14 @@
 	<key>LSApplicationQueriesSchemes</key>
 	<array>
 		<string>mailto</string>
```

Comment:
CanOpenUrl is also called for `ms-outlook://` and `googlegmail://` schemes. Should they be added into this list too?